cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,293 @@
+# Table of contents: 
+[Dynamic Objects (Only Supported in JS Objects)](#dynamic-objects-only-supported-in-js-objects)
```

Comment:
table of contents is not arranged properly. does all heading come under **Supported Dynamic Glue Codes** ?

Reply:
No, corrected the table